frontend/simple/views/GroupDashboard.vue

```diff
@@ -0,0 +1,36 @@
+<script>
+export default {
+  name: 'GroupDashboard',
+  computed: {
+    group () {
+      return this.$store.getters.currentGroup
+    }
+  },
+  methods: {
+    invite (groupId) {
+      this.$store.commit('setCurrentGroupId', groupId)
+      this.$router.push({path: '/invite'})
+    }
+  }
+}
+</script>
+
+<template>
+  <section class="container section">
+    <p class='is-pulled-right'>
+      <a class="button is-primary" @click.prevent="invite(group.id)">
+        <span class="icon is-small"><i class='fa fa-plus mr-sm'></i></span>
+        <span><i18n>Invite members</i18n></span>
+      </a>
+    </p>
+    <h1 class="title is-1 mb-none">{{ group.groupName }}</h1>
```

Comment:
I see you're using `mb-none` and `mr-sm`... are these existing classes in `master`? I haven't merged @danawoodman's PR yet because there's an issue with it.